shared/rocroller/lib/include/rocRoller/KernelGraph/ControlToCoordinateMapper.hpp

```diff
@@ -75,6 +75,29 @@ namespace rocRoller::KernelGraph
             return a.id < b.id;
         }
 
+        struct UnrollStride
+        {
+            std::string unrollStride;
+            int         unrollDimension;
+
+            bool operator==(UnrollStride const& other) const
+            {
+                return this->unrollStride == other.unrollStride
+                       && this->unrollDimension == other.unrollDimension;
+            }
```

Comment:
Is this equivalent to the provided default?
```suggestion
            bool operator==(UnrollStride const& other) const = default;
```

Reply:
Even better would be `auto operator<=>(UnrollStride const& other) const = default;` if that works as well.

Reply:
Updated in [383ef6f](https://github.com/ROCm/rocm-libraries/pull/1387/commits/383ef6f347f60ae0c7fda9aa0f313a1b02d07259).